pkg/sdk/go/sdk.go

```diff
@@ -138,6 +148,30 @@ type SDK interface {
 	// DeleteThing removes existing thing.
 	DeleteThing(id, token string) error
 
+	// CreateUsersGroup creates new group and returns its id.
+	CreateUsersGroup(group Group, token string) (string, error)
+
+	// DeleteUsersGroup deletes users group.
+	DeleteUsersGroup(id, token string) error
+
+	// UsersGroups returns page of users groups.
+	UsersGroups(token string, offset, limit uint64, name string) (UsersGroupsPage, error)
+
+	// UsersGroup returns users group object by id.
+	UsersGroup(id, token string) (Group, error)
+
+	// AssignUserToGroup assigns user to a group
+	AssignUserToGroup(userID, groupID, token string) error
+
+	// RemoveUserFromGroup
+	RemoveUserFromGroup(userID, groupID, token string) error
+
+	// ListUsersForGroup
+	ListUsersForGroup(groupID, token string, offset, limit uint64) (UsersPage, error)
```

Comment:
This naming is not consistent with the rest. I would use `CreateGroupUser`, `RemoveGroupUser` and `GroupUsers`